### ISSUE TITLE
Timer Role is Missing Implicit Values Row in Table

### ISSUE THREAD

**Issue by Epigenetic** [w3c/aria#1904]
# Describe the change
The timer role's characteristics table is missing the implicit values row, however it does have an implicit value for aria-live.

# Link to the version of the specification or documentation you were looking at
https://w3c.github.io/aria/#timer

Link to documentation: https://w3c.github.io/aria/#timer

Does the issue exists in [the editors draft](https://w3c.github.io/aria/) (the editors draft is the most recent draft of the specification)? Yes
